submissions/chabuChabu/database/trips.sql

```diff
@@ -0,0 +1,37 @@
+/*The following query is used to create a table for storing passanger details*/
+use trips;
+create table passangers(
+id int not null auto_increment,
+fullname varchar(100) not null,
+sex varchar(10) not null,
+age varchar(11),
+primary key (id)
+);
+
+/*The following query is used to create a table for storing trip details*/
+/*It references the passanger table  via passanger ID*/
+create table trip_details(
+id int not null auto_increment,
+passanger_class int(11) not null,
+ticket_number varchar (50) not null,
+trip_fare decimal(10,2)not null,
+cabin varchar (50),
+parents int (11),
+children int (11),
```

Comment:
This supposes to be `no_of_parent_or_children` as a single column. Not having `parents` and `children` as a separate column